addons/hr_timesheet/views/project_task_views.xml

```diff
@@ -38,7 +38,7 @@
                                 You cannot log timesheets on this project since it is linked to an inactive analytic account. Please change this account, or reactivate the current one to timesheet on the project.
                             </div>
                         </group>
-                    <field name="timesheet_ids" mode="tree,kanban" attrs="{'invisible': [('analytic_account_active', '=', False)]}" context="{'default_project_id': project_root_id, 'default_name':''}">
+                    <field name="timesheet_ids" mode="tree,kanban" attrs="{'invisible': [('analytic_account_active', '=', False)]}" context="{'default_project_id': project_id, 'default_name':''}">
```

Comment:
I tested to be sure we will have the project set after the creation and indeed the `_compute_project_id` will do the job, since the task displayed in the form view will automatically be set. 🙂